### DISCUSSION TITLE
Solidity Shell

### DISCUSSION THREAD

**Discussion by leomileadover** [Cyfrin/foundry-full-course-cu#1748] [Q&A] [answered]
I swear Patrick mentioned a Solidity shell in one of his videos.
Something which works like Python's ipython where it pops open an interactive shell that you can type in solidity commands with.

Does anyone remember what that command/package is?

**Comment by footprns** [comment, marked as answer]
Hi @leomileadover , is it `forge`, `cast`, `anvil` and `chisel` ?

**Reply by leomileadover** [reply]
Chisel was the one i had in mind! Thousand thanks, it was driving me mad!! haha

**Reply by EngrPips** [reply]
Interesting.
